components/sup/src/manager.rs

```diff
@@ -1370,14 +1377,16 @@ impl Manager {
     /// # Locking (see locking.md)
     /// * `MemberList::entries` (read)
     /// * `RumorStore::list` (write)
-    fn restart_elections_rsw_mlr(&mut self, feature_flags: FeatureFlag) {
-        self.butterfly.restart_elections_rsw_mlr(feature_flags);
+    /// * `ManagerServices::inner` (read)
+    /// * `ManagerServices::inner` (write)
+    fn restart_elections_rsw_mlr_msr(&mut self, feature_flags: FeatureFlag) {
```

Comment:
You only need the read annotation here right?

Reply:
Yup, good catch